zipkin-sampler/src/main/scala/com/twitter/zipkin/sampler/AdaptiveSampler.scala

```diff
@@ -58,6 +85,24 @@ trait AdaptiveSampler { self: App with ZooKeeperClientFactory =>
     5.minutes,
     "Amount of time to see outliers before updating sample rate")
 
+  val zkServerLocations = flag(
```

Comment:
this is just folding in the ZK trait. I removed the trait as we don't want to encourage its use.